src/react/utils.js

```diff
@@ -665,12 +662,10 @@ export function getProperty(
       return realm.intrinsics.undefined;
     }
     let elements = object.values.getElements();
-    invariant(elements);
-    if (elements.size > 0) {
-      object = Array.from(elements)[0];
-    } else {
-      // intentionally left in
-      invariant(false, "TODO: should we hit this?");
+    invariant(elements.size === 1, "TODO: deal with multiple elements");
```

Comment:
@trueadm I don't know what this is supposed to do but currently it is only dealing with one possible value here. This is broken when it can be more than one possible object. I'm not sure what this is supposed to do in that case.

Reply:
In principle, the `getProperty` method is only meant to be used on Objects that can only ever have one value as they're React objects that we control. So this invariant is fine with me.